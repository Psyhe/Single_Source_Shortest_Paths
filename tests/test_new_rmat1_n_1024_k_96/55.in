1024 605 615
606 642 111
608 129 25
608 1 75
608 52 70
608 66 115
608 32 76
608 144 79
608 772 66
608 576 246
608 1 233
608 416 87
608 0 117
608 328 14
608 32 137
608 66 165
608 528 51
608 0 229
608 592 51
608 9 172
608 300 202
608 384 238
608 2 136
608 680 225
608 258 90
608 544 236
608 32 235
608 128 26
608 2 10
608 6 144
608 522 116
608 385 196
608 149 146
608 564 25
608 914 140
608 3 17
608 24 78
608 256 254
608 451 75
608 514 252
608 34 165
608 8 46
608 168 147
608 434 107
608 136 26
608 133 238
609 64 17
609 128 177
609 24 230
609 32 205
609 512 203
609 350 2
610 290 90
610 517 192
610 64 93
610 97 230
610 41 170
610 771 0
610 8 58
610 2 243
610 768 124
610 912 121
610 134 58
610 1 15
610 388 157
610 74 185
610 576 239
611 0 24
611 8 154
611 192 234
611 48 55
612 401 158
612 161 191
612 516 45
612 513 213
612 17 28
612 0 63
612 280 38
612 145 134
612 0 206
613 530 226
614 16 79
614 8 198
614 1 234
614 144 187
615 132 136
