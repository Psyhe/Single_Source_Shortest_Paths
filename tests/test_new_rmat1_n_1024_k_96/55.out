1e+18
134
1e+18
15
17
17
24
35
243
84
139
